staging/src/k8s.io/sample-apiserver/pkg/admission/plugin/banflunder/admission.go

```diff
@@ -80,5 +88,7 @@ func (d *disallowFlunder) Validate() error {
 
 // New creates a new ban flunder admission plugin
 func New() (admission.Interface, error) {
-	return &disallowFlunder{}, nil
+	return &disallowFlunder{
+		Handler: admission.NewHandler(admission.Create),
```

Comment:
as mentioned in my previous PR names are immutable. I think it is safe to assume  that the plugin should only support `admission.Create` operation ?